Add associations improvements (#537)
akeneo/pim-helpcenter: content/md/products-super-power/product-associations.md

![View the associated products](../img/Products_ProductsAssociated21.png)

:::info

> The number of associations for a product or a product model is displayed in total (after the `Associations` tab label) and by association type (in the `Association type` list).

Reviewer: ```suggestion
The total number of associations is displayed per product or product model (the figure is visible after the `Associations` tab label) and per association type (figure visible in the `Association type` list).
```
Did I keep the global sense here?

Author: Yes nickel :)
